app/src/main/java/org/oppia/android/app/databinding/TextInputLayoutBindingAdapters.java

```diff
@@ -15,4 +16,14 @@ public static void setErrorMessage(
   ) {
     textInputLayout.setError(errorMessage);
   }
+
+
+  /** Binding adapter for setting the text of an [AutoCompleteTextView]. */
+  @BindingAdapter({"selection", "filter"})
+  public static void setSelection(
```

Comment:
Any tests for this?

Reply:
I've created a new test suite. Seems like the test file presence check does not check .java files.